typeid/typeid-js/src/unboxed/typeid.ts

```diff
@@ -78,6 +78,10 @@ export function fromString<T extends string>(
     }
   }
 
+  if (!s) {
```

Comment:
Is this function meant to parse ids missing `_`  anyway?

It seems like line 67 should be:
```ts
if (underscoreIndex === -1)
  throw new Error()
}
```

I see the docbloc, but why does this function allow suffix only? That's not a valid typeid.

Reply:
I think prefix is not required for a typeId. @loreto is that the case?

Reply:
You are correct. The spec says prefix may be empty and _ omitted: https://github.com/jetify-com/typeid/tree/main/spec

my bad.